Makefile

```diff
@@ -192,33 +194,33 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 endif
 
 i18n-extract: ## Extract strings for translation from the source code
-	env GO111MODULE=off go get -u github.com/mattermost/mattermost-utilities/mmgotool
-	$(GOPATH)/bin/mmgotool i18n extract
+	cd tools && $(GO) install -mod=readonly github.com/mattermost/mattermost-utilities/mmgotool && cd ..
+	mmgotool i18n extract
 
 store-mocks: ## Creates mock files.
-	env GO111MODULE=off go get -u github.com/vektra/mockery/...
-	$(GOPATH)/bin/mockery -dir store -all -output store/storetest/mocks -note 'Regenerate this file using `make store-mocks`.'
+	cd tools && $(GO) install -mod=readonly github.com/vektra/mockery/cmd/mockery && cd ..
```

Comment:
1/5 that it would be nice to wrap this in a function, which we reuse every time we want a tool.

Reply:
I was thinking about this whether Makefiles support functions or not, but now that you mention it, I will look into it and send a PR later :wink: